source/renderer/app/utils/shelleyTrezor.ts

```diff
@@ -77,19 +81,20 @@ export type TrezorVotingDataType = {
   votingKey: string;
   nonce: string;
 };
+
 export const prepareTrezorAuxiliaryData = ({
   votingKey,
   nonce,
 }: TrezorVotingDataType) => ({
-  catalystRegistrationParameters: {
-    votingPublicKey: votingKey,
+  cVoteRegistrationParameters: {
```

Comment:
How come this was shipped as a patch (9.0.2 -> 9.0.8) 🤯

Reply:
yeah, it is weird but it is shipped as a patch :(